packages/autocomplete-js/src/render.tsx

```diff
@@ -109,7 +109,9 @@ export function renderPanel<TItem extends BaseItem>(
         </div>
       )}
 
-      {items.length === 0 && source.templates.noResults && state.query ? (
+      {!items.length &&
```

Comment:
We use this syntax in the codebase.

```suggestion
      {items.length === 0 &&
```

Reply:
Fixed in https://github.com/algolia/autocomplete/pull/573/commits/bfb80ebd4d9bf719923e5f21b1aa1e3264c412ea